content/2_Bronze/Intro_Complete.mdx

```diff
@@ -167,6 +188,63 @@ Rounding suffices (`(int)round(pow(max_dist,2))`), but the takeaway is that you
 should stick with integers whenever possible.
 
 </CPPSection>
+	
+<JavaSection>
+
+The following solution correctly stores the maximum distance in a floating point
+variable.
+
+```java
+import java.util.*;
+import java.io.*;
+
+public class MaximumDistance2 {
+	public static void main (String[] args) {
+		Kattio io = new Kattio();
+
+		int n = io.nextInt();
+		int[] x = new int[n];
+		int[] y = new int[n];
+
+		for (int i = 0; i < n; i++) {
+			x[i] = io.nextInt();
+		}
+		for (int i = 0; i < n; i++) {
+			y[i] = io.nextInt();
+		}
+
+		double maxSquare = 0; // storing the current maximum
+		for (int i = 0; i < n; i++) {  // for each first point
+			for(int j = i + 1; j < n; j++){ // for each second point
+				int dx = x[i] - x[j];
+				int dy = y[i] - y[j];
+				int  square = dx * dx + dy * dy;
+				// if the square of the distance between the two points is greater than
+				// our current maximum, then update the maximum
+				maxSquare = Math.max(maxSquare, Math.sqrt(square));
+			}
+		}
+		io.println((int) Math.pow(maxSquare, 2));
+		io.close();
+	}
+}
+
+```
+
+However, it still fails on the following test case (it outputs 12, but the
```

Comment:
I would like it if the fact that it was incorrect was _more_ emphasized (probably bolded or in a spoiler)